tasks/CentOS/main.yml

```diff
@@ -4,18 +4,18 @@
 
   - include_vars: "{{ ansible_distribution }}.yml"
 
+  - name: Set sensu_pkg_version {{ ansible_distribution }} override
```

Comment:
This works because the var is set to a 'none' values in defaults/main.yml.  The task when conditional is triggered on variable loaded from defaults/main.yml, but the lookup for the set_fact value is overridden and pulled in from the include_vars tasks above this one. 

This was the only way I was able to set a ''default'' value based on something like the OS but still allow it to be overridable via group_vars or host_vars

So if you set the sensu_pkg_version value in defaults/main.yml or override it via host_vars or group_vars it not be none.  Thus not loading the ''default'' from vars/Centos.yml

FreeBSD already had the 'constant' variable set in /vars/FreeBSD.yml.  I call these constant variables as you can not override them easily within an inventory structure, group_vars or host_vars

It was added just to CentOS for our current needs, we had issues getting the testing for some of the distros but were able to use our in-house testing for a lot of these.